protos/bc.proto

```diff
@@ -24,24 +24,32 @@ message RpcTransaction {
 message RpcFeedTransaction {
     string owner_addr = 1;
     string feed_addr = 2;
-    string amount = 3;
-    string tx_fee = 4;
-    string private_key_hex = 5;
+    string data_type = 3;
+    string data_length = 4;
+    string data = 5;
+    string amount = 6;
+    string tx_fee = 7;
+    string tx_part = 8;
+    string tx_panel = 9;
+    string tx_nonce = 10;
+    string minimum_distance = 11;
+    string private_key_hex = 12;
 }
 
 message RpcUpdateFeedTransaction {
```

Comment:
I won't copy the comments from `RpcFeedTransaction` message, but same applies here for `RpcUpdateFeedTransaction`